pages/doc/opentracing_to_opentelemetry_migration.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Migrate from OpenTracing to OpenTelemetry
+keywords: data, distributed tracing
+tags: [tracing]
+sidebar: doc_sidebar
+permalink: opentracing_to_opentelemetry_migration.html
+summary: Migrate your Java application from OpenTracing to use OpenTelemetry.
+---
+
+OpenTracing is deprecated. ([OpenTracing](https://opentracing.io/) and [OpenCensus](https://opencensus.io/) have merged to form [OpenTelemetry](https://opentelemetry.io/).) To send trace data to Tanzu observability, use OpenTelemetry.
```

Comment:
Seems I've missed this "Tanzu observability" during the review. Can we fix it to Tanzu Observability before merging?